control/flags.go

```diff
@@ -19,19 +19,21 @@ limitations under the License.
 
 package control
 
-import "github.com/codegangsta/cli"
+import (
+	"fmt"
+
+	"github.com/codegangsta/cli"
+)
 
 var (
-	flNumberOfPLs = cli.IntFlag{
+	flNumberOfPLs = cli.StringFlag{
```

Comment:
I certainly agree that we don't switch CLI libraries.  I think the other option here is to use lookups for envs, and retrieve them via `os.Getenv` rather than from the CLI library, because, as you said, Tom, you can't get the CLI `env`s from the library itself.